src/identities/identities.controller.ts

```diff
@@ -622,4 +623,58 @@ export class IdentitiesController {
 
     return new GroupedInstructionModel(result);
   }
+
+  @ApiOperation({
+    summary: 'Rotate Primary Key',
+    description:
+      'Creates an Authorization to rotate primary key of the signing Identity by the `targetAccount`.',
```

Comment:
For rotate is it the current key becomes a secondary key? I think it would be nice to include the details if that's the case.

```suggestion
      'Creates an Authorization to rotate primary key of the signing Identity by the `targetAccount`. When accepted the current primary key will become a secondary key',
```

If it becomes unattached then:
```suggestion
      'Creates an Authorization to rotate primary key of the signing Identity by the `targetAccount`. When accepted the current primary will be unlinked from the Identity,
```